pycbc/results/scatter_histograms.py

```diff
@@ -359,8 +364,8 @@ def create_marginalized_hist(ax, values, label, percentiles=None,
             ax.axvline(expected_value, color=expected_color, lw=1.5, zorder=2)
     if title:
         values_med = numpy.median(values)
-        values_min = values.min()
-        values_max = values.max()
+        values_min = numpy.percentile(values, 5.)
+        values_max = numpy.percentile(values, 95.)
```

Comment:
So this is hard-coded to quote the middle 90% confidence interval always? If someone wants to read off the 68% confidence interval (a common % quoted by astronomers) that can't be done, right? Also, the percentiles plotted will not always be related to the numbers quoted in the title; if someone plots the 16 and 84 percentile, they will read the 90% credible interval not the 68% CI. That can be confusing.

Reply:
Ok, I've now changed it so that it quotes the percentile halfway between the max/min of the `percentiles` +/- max/min of the `percentiles`. It will default to 50 +/- 95/5 if no percentiles were given. I chose average of the percentiles for the median value because just quoting the median will break if you gave percentiles that did not encompass the median.

Aside: you say in the astronomy they like to quote the 68% confidence interval. But do they quote the median +/- 84/16th percentile or the mean +/- 1 sigma? If the latter, you still can't use the quoted values in the title, since the mean is not a percentile.

Reply:
You're right, they quote 68% credible interval as the mean +/- 1.0 sigma. Thanks for correcting that. & the new changes relating the percentile lines plotted with the quoted values look good to me.